judge/models/contest.py

```diff
@@ -53,6 +54,11 @@ class Meta:
         verbose_name_plural = _('contest tags')
 
 
+class ContestManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().annotate(_now=Now())
```

Comment:
I am unconvinced whether SQL `Now()` is the best approach. Hypothetically, if we were to run separate queries for past contests for pagination purposes, the time would be desynchronized between different queries. If instead, you used `timezone.now()`, we could do `Contest.objects.all()` and then filter separately, and they will all have the same time.